### ISSUE TITLE
Hello, Can you provide a specific operational process?

### ISSUE THREAD

**Issue by yuluoxingchen111** [tudelft3d/Building-PCC-Building-Point-Cloud-Completion-Benchmarks#1]
Hello, may I ask what the specific operation is? For example, if I want to run the PoinTr model in the code you provided, do I only need to refer to the original way the PoinTr model runs without running any other commands? Can you provide a specific operational process? (I have downloaded the PoinTr dataset and pre trained model)

**Comment by WeixiaoGao** [comment]
Yes, please refer to the instructions provided with the original PoinTr. You should use the version of PoinTr we have supplied, which includes our data configuration files.

**Comment by yuluoxingchen111** [comment]
Excuse me, when I made my building point cloud data into a PCN dataset and trained it in the original PoinTr with KITTI configuration, the result was very poor, and there was no significant change after modifying the parameters. Is this situation because my data was not well made? (I directly replaced the Cars in the original model KITTI dataset with self-made data, and created new. yaml configuration files and JSON files.)
